src/components/chip-set/chip.types.ts

```diff
@@ -44,4 +44,9 @@ export interface Chip<T = any> {
      * Value of the chip.
      */
     value?: T;
+
+    /**
+     * The value of the badge. Only valid for `filter`.
+     */
+    badgeValue?: number;
```

Comment:
If we rename this to just `badge`, we leave more room to add new features to this in the future. Right now we just allow a single value, but later we might want to add support for changing the color etc. e.g. by doing something like this

```ts
interface Chip {
    badge?: number | BadgeConfig;
}

const chip = {
    badge: {
        value: 4,
        backgroundColor: 'red'
    }
};
```

What do you think? :slightly_smiling_face:

Reply:
ah, yes sounds good :)